Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -100,12 +111,31 @@ final class PledgeCTAContainerView: UIView {
 
 // MARK: - Styles
 
-private let projectStateButtonStyle: ButtonStyle = { (button: UIButton) in
-  button
-    |> roundedStyle(cornerRadius: 12)
-    |> UIButton.lens.titleColor(for: .normal) .~ .white
-    |> UIButton.lens.titleLabel.font .~ .ksr_headline(size: 15)
-    |> UIButton.lens.layer.borderWidth .~ 0
-    |> UIButton.lens.titleEdgeInsets .~ .init(topBottom: Styles.grid(1), leftRight: Styles.grid(2))
-    |> (UIButton.lens.titleLabel .. UILabel.lens.lineBreakMode) .~ .byWordWrapping
+private func adaptableStackViewStyle(_ isAccessibilityCategory: Bool) -> (StackViewStyle) {
+  return { (stackView: UIStackView) in
+    let axis: NSLayoutConstraint.Axis = (isAccessibilityCategory ? .vertical : .horizontal)
+    let spacing: CGFloat = (isAccessibilityCategory ? Styles.grid(1) : 0)
+
+    return stackView
+      |> \.axis .~ axis
+      |> \.spacing .~ spacing
+  }
+}
+
+private func projectStateButtonStyle(
+  _ isAccessibilityCategory: Bool, amountAndRewardTitleStackViewIsHidden: Bool
+) -> (ButtonStyle) {
+  return { (button: UIButton) in
+    let lineBreakMode: NSLineBreakMode = isAccessibilityCategory || amountAndRewardTitleStackViewIsHidden
```

Comment:
| Before 🐛 | After 🦋 |
| --- | --- |
| <img width="460" alt="Screen Shot 2019-06-25 at 5 50 09 PM" src="https://user-images.githubusercontent.com/387596/60143878-3359c600-9775-11e9-9b19-a583336b4148.png"> | <img width="460" alt="Screen Shot 2019-06-25 at 5 49 02 PM" src="https://user-images.githubusercontent.com/387596/60143885-3a80d400-9775-11e9-8b5c-19e3466e7dc9.png"> |

This hopefully fixes the issue of not using `wordWrap` when the button can fit properly. Maybe there's a way around with `compression resistance` or `content hugging` priorities but I didn't have much luck..Cc: @justinswart 🤔

Reply:
I think we want to avoid compression and hugging priorities if possible, seems like a good enough solution to change the `lineBreakMode`